### PR TITLE
switch from deb to tarball

### PR DESCRIPTION
**Pull request by eszlari**
https://github.com/flathub/com.visualstudio.code/issues/2
https://github.com/flathub/com.visualstudio.code/issues/8

### CODE REVIEW

File: com.visualstudio.code.appdata.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		</screenshot>
 	</screenshots>
 	<releases>
-    	<release version="1.18.0" date="2017-11-08"/>
```
Comment (Barthalion) [inline com.visualstudio.code.appdata.xml]: Don't remove the old release (and move the latest one to the top to make appstream-compose happy).

**Comment by Barthalion** [comment]
bot, build

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
Does the tarball not ship with a desktop file? Not a massive issue, but the less that has to be maintained the better.

**Comment by eszlari** [comment]
> Does the tarball not ship with a desktop file? Not a massive issue, but the less that has to be maintained the better.

https://github.com/Microsoft/vscode/issues/41294

**Comment by nedrichards** [comment]
thanks, lgtm

**Pull request merged**